### PR TITLE
TestCaseRun to TestExecution leftovers

### PR DESCRIPTION
**Pull request by asankov**

### CODE REVIEW

**Comment by atodorov** [comment]
Also fix failures!

File: tcms/testcases/views.py [hunk 1]
```diff
@@ -511,27 +511,26 @@ def get_context_data(self, **kwargs):
...
-            'test_case_run': case_run,
-            'comments_count': len(caserun_comments),
-            'caserun_comments': caserun_comments,
-            'caserun_logs': case_run.history.all(),
-            'test_status': caserun_status,
+            'test_case_run': execution,
+            'comments_count': len(execution_comments),
+            'caserun_comments': execution_comments,
+            'caserun_logs': execution.history.all(),
+            'test_status': execution_status,
```
[the remark this answers is not in the document]
Comment (asankov) [reply tcms/testcases/views.py]: Ok, but I suggest we leave the template name as it is `get_details_case_run.html` as this is legacy and at some point will be refactored into Patternfly and deleted anyway

File: tcms/locale/sl_SI/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1362,14 +1362,6 @@ msgstr "Izberi vse / Odznači vse"
 msgid "Expand all cases"
 msgstr "Razširi vse"
 
-#: tcms/templates/run/table_caseruns.html:15
-msgid "Case Run ID"
-msgstr "ID testiranja"
```
Comment (atodorov) [inline tcms/locale/sl_SI/LC_MESSAGES/django.po]: There is no "Execution ID" string in this po file. Better drop all modifications from .po files and redo them at the end of this PR or after this gets merged.

Comment (asankov) [reply tcms/locale/sl_SI/LC_MESSAGES/django.po]: Done. I suppose this way is a lot easier, but the other commits are not atomic

**Comment by asankov** [comment]
Build with MariaBD seems to be failing:
```
django.db.utils.OperationalError: (2003, 'Can\'t connect to MySQL server on \'127.0.0.1\' (111 "Connection refused")')
Makefile:27: recipe for target 'test' failed
```
I see that you have recently made some changes to the configurations, but I am on the latest master. Can you give some advise?

**Comment by atodorov** [comment]
Testing with MariaDB started failing today. Even previous builds which were passing are now failing. I've filed an issue with Travis CI Support and waiting for the response.  Until then merging is blocked.

**Pull request merged**